.github/workflows/paralleltest-docker.yml

```diff
@@ -0,0 +1,91 @@
+name: Run AQAvit Paralleltest docker
+
+on:
+  workflow_dispatch: # Allows the job to be manually triggered
+    inputs:
+      customizedSdkUrl:
+         description: SDK URL
+         required: true
+      testImage:
+        description: Native Libs URL
+        required: true
+      aqatest-repo:
+        description: adoptium/aqa-tests repository
+        required: true
+        type: string
+        default: 'adoptium/aqa-tests:v0.9.6-release'
+      containerRegistry:
+        description: Container registry
+        required: true
+
+jobs:
+  setup-parallel-docker:
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ inputs.containerRegistry }}
+      env:
+        USE_TESTENV_PROPERTIES: true
+      options: --cpus 2
+      credentials:
+        username: ${{ secrets.ACR_TOKEN_NAME }}
+        password: ${{ secrets.AZURE_ACR_TOKEN_PASS1 }}
+    strategy:
+      fail-fast: true
+      matrix:
+        suite: [functional, openjdk, system, perf]
+    steps:
+
+      - uses: adoptium/run-aqa@v2.0.1
+        with:
+          jdksource: 'customized'
+          customizedSdkUrl: ${{ inputs.customizedSdkUrl }}
+          aqa-testsRepo: ${{ inputs.aqatest-repo }}
+          build_list: ${{ matrix.suite }}
+          target: ${{ matrix.suite }} # parallellist to generate
+          run_parallel: true
+          num_machines: 4
+
+      - name: Archive parallelList - ${{ matrix.suite }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ matrix.suite }}-parallelList.mk
+          path: ${{ github.workspace }}/aqa-tests/TKG/parallelList.mk
+
+  run-aqa-parallel-docker:
+    runs-on: ubuntu-latest
+    needs: setup-parallel-docker
+    container:
+      image: ${{ secrets.CONTAINER_REGISTRY }}
+      env:
+        USE_TESTENV_PROPERTIES: true
+      options: --cpus 2
+      credentials:
+        username: ${{ secrets.ACR_TOKEN_NAME }}
+        password: ${{ secrets.AZURE_ACR_TOKEN_PASS1 }}
+    strategy:
+      fail-fast: false
+      matrix:
+        test_list: ['testList_0', 'testList_1', 'testList_2', 'testList_3'] # numlist is hardcoded...for now

```

Comment:
Is there a work item/GitHub issue we can link to here, to show intent to fix later?

Reply:
Internal ticket created (I dont seem to be able to create a GH issue here.